Automattic/sensei
# Fix being unable to set start data in Learner Management

Fixes #1933.

## Testing

1. Go to Sensei > Learner Management.
2. Select a course that has more than 1 learner.
3. Change the start date for a learner, then click the _Edit Start Date_ button. The start date should be updated after the page auto-refreshes.
4. Repeat step 3 for other learners. Ensure that you test more learners than just the first one in the list, as changing the start date for the first learner worked prior to this PR.
